classes/classroom.rb

```diff
@@ -0,0 +1,19 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] Good job implementing the `has-many` relationship between the `students` and `classroom`. However, the `belongs-to` side of the relationship is missing, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20belongs%2Dto%20side%20(a%20student%20belongs%20to%20a%20classroom).) requirement. This can be done in the `student` class.